inlang/source-code/sdk/src/v2/types/project-settings.ts

```diff
@@ -59,7 +59,11 @@ const InternalProjectSettings = Type.Object({
 			],
 		}
 	),
-	lintConfig: Type.Array(LintConfig),
+	lintConfig: Type.Array(LintConfig, {
```

Comment:
@martin-lysk I added title and description for the array. (It's displayed in the settings view)